tests/unit/models/test_base_model.py

```diff
@@ -397,14 +504,18 @@ def test_from_xrpl(self):
             for test in fixtures_json["transactions"]:
                 x_json = test["xjson"]
                 r_json = test["rjson"]
-                with self.subTest(json=x_json):
+                with self.subTest(json=x_json, use_json=False):
```

Comment:
The `use_json` parameter in these function calls are purely for debugging purposes right? I flipped the `True/False` values in these three tests, it did not affect the tests in any way.

<img width="966" alt="image" src="https://github.com/XRPLF/xrpl-py/assets/21219765/0eada3c1-183e-4222-83f6-b8706fe42b78">

Reply:
Yes. Note how the `use_json=True` test is the only one that uses `json.dumps`. I did that for better code coverage.

Reply:
Hmm right. In that test case, `json.dumps` returns a JSON string, rather than a Python object. This forces `Transaction.from_xrpl` to use a different codepath, namely the first type in the Union: `def from_xrpl(cls: Type[BM], value: Union[str, Dict[str, Any]]) -> BM:`

good observation